icu4j/main/core/src/main/java/com/ibm/icu/message2/TextSelectorFactory.java

```diff
@@ -26,7 +27,23 @@ private static class TextSelector implements Selector {
          * {@inheritDoc}
          */
         @Override
-        public boolean matches(Object value, String key, Map<String, Object> variableOptions) {
+        public List<String> matches(
+                Object value, List<String> keys, Map<String, Object> variableOptions) {
```

Comment:
What's wrong with it?

It is 8 spaces. The indent is 4, and a line wrap should use 2 indents.
And I can't unwrap because the total line length is 108.

Also, this is wrapped by [`diffplug/spotless`](https://github.com/diffplug/spotless/tree/main/plugin-maven) using Google AOSP rules (Google rules but with 4 spaces indent instead of 2, because the rest of ICU uses 4).